.github/workflows/benchmark-comparison.yml

```diff
@@ -0,0 +1,55 @@
+name: Benchmark comparison
+on:
+  workflow_dispatch:
+    inputs:
+      bench:
+        description: 'Benchmarks to run'
+        required: false
+        default: '.'
+      parallelism:
+        description: 'Number of parallel benchmarks to run'
+        required: false
+        default: 5
+      duration:
+        description: 'Duration of each benchmark'
+        required: false
+        default: '10s'
+      count:
+        description: 'Number of times to run each benchmark '
+        required: false
+        default: 1
+  pull_request:
+    types: [ assigned, opened, synchronize, reopened, labeled ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+  BenchmarkCompare:
+    runs-on: "github-001"
```

Comment:
Why only on github-001 ?

Reply:
No specific reason, i just copy/paste from the original workflow.